src/main/perf/facets/BenchmarkFacets.java

```diff
@@ -192,9 +208,14 @@ public IndexSearcher newSearcher(IndexReader reader, IndexReader previous) {
                 reportPercentDifference(totalTaxoGetTopDimsTimeMS / (double) i, totalSSDVGetTopDimsTimeMS / (double) i);
                 System.out.println("");
 
-                System.out.println("Time (ms) taken to get top children of \"address.taxonomy/TX\" children for taxonomy: " + totalTaxoGetChildrenTimeMS / (double) i);
-                System.out.println("Time (ms) taken to get top \"address.sortedset/TX\" children for SSDV: " + totalSSDVGetChildrenTimeMS / (double) i);
-                reportPercentDifference(totalTaxoGetChildrenTimeMS / (double) i, totalSSDVGetChildrenTimeMS / (double) i);
+                System.out.println("Time (ms) taken to get top children of \"address.taxonomy/TX\" children for taxonomy: " + totalTaxoGetTopChildrenTimeMS / (double) i);
+                System.out.println("Time (ms) taken to get top \"address.sortedset/TX\" children for SSDV: " + totalSSDVGetTopChildrenTimeMS / (double) i);
+                reportPercentDifference(totalTaxoGetTopChildrenTimeMS / (double) i, totalSSDVGetTopChildrenTimeMS / (double) i);
+                System.out.println("");
+
+                System.out.println("Time (ms) taken to get all children of \"address.taxonomy/TX\" children for taxonomy: " + totalTaxoGetAllChildrenTimeMS / (double) i);
```

Comment:
So I think this statement is wrong as you are calling this: `result = taxoFacets.getAllChildren(TAXO_FIELD_NAME);` and not specifying a field name of TX. With that being said, I think a more rigorous test of `getAllChildren()` could be done by specifying a path that has more children than just the field name (that will give you just the dims and I don't think there are too many of those). If possible, could you explore a bit and see if there are any paths you can find that give you a large amount of children and maybe use that as the test?

Reply:
Thanks for catching this! Sorry, I just copied and edited it from the previous statement, and the path name  \"address.taxonomy/TX\" stated in this print statement should be removed in order to avoid confusion. 

I also tried running benchmark for taxoFacets.getAllChildren(TAXO_FIELD_NAME, "TX") and ssdvFacets.getAllChildren(SSDV_FIELD_NAME, "TX"), and observed a significant reduced time in populating all children results compared to taxoFacets.getAllChildren(TAXO_FIELD_NAME) and ssdvFacets.getAllChildren(SSDV_FIELD_NAME)

```
Results after iteration 20
Time (ms) taken to instanciate FastTaxonomyFacetCounts: [2337.3920009](tel:23373920009)
Time (ms) taken to instanciate SSDVFacetCounts: 5411.58398475
SSDV is 131.5% slower than Taxonomy
Time (ms) taken to get all dims for taxonomy: 0.15421010000000002
Time (ms) taken to get all dims for SSDV: 0.06380064999999999
Taxonomy is 141.7% slower than SSDV
Time (ms) taken to get top dims for taxonomy: 0.07789034999999998
Time (ms) taken to get top dims for SSDV: 0.04125235000000001
Taxonomy is 88.8% slower than SSDV
Time (ms) taken to get top children of "address.taxonomy/TX" children for taxonomy: 0.1363089
Time (ms) taken to get top "address.sortedset/TX" children for SSDV: 0.17233800000000002
SSDV is 26.4% slower than Taxonomy
Time (ms) taken to get all children of "address.taxonomy/TX" children for taxonomy: 0.9590940499999998
Time (ms) taken to get all "address.sortedset/TX" children for SSDV: 0.8944382000000003
Taxonomy is 7.2% slower than SSDV
```
I will look into dataset and see if I can find a child path that with more children than the "TX" path under the "SSDV_FIELD_NAME" and "TAXO_FIELD_NAME" dims OR more children than the total number children under these two dims. Thanks for the suggestion!

Reply:
I've looked at the allChildren and topChildren results of `SSDV/TAXO_FIELD_NAME`. The `TX` path is still the best candidate for `getAllChildren` since it has most children ( > 250). This also makes sense geographically. I've updated the path in the second commit. Please let me know if there is any question. Thank you so much!

Reply:
Code LGTM, thanks for taking the time to look into the data!